sdk/servicebus/service-bus/src/queueClient.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import * as Long from "long";
+import Long from "long";
```

Comment:
What warning did this fix?

Reply:
This comes from us importing Long using both "import Long from "long" " and "import * as Long from "long" "

What you get in the merged .d.ts file is a mix of import names, including a rename of Long to Long2. I figured, we didn't want Long2 as a separate type, and in general for our .d.ts files we'd want one way of importing a symbol to keep things clean.